### ISSUE TITLE
Net not Learning when Batch size > 1

### ISSUE THREAD

**Issue by CataRay** [CataRay/tf_Medical-Image-Segmentation#1]
When the net is trained with minibatch size of 1, I get fairly good results:
![screen shot 2018-04-04 at 2 31 48 pm](https://user-images.githubusercontent.com/7873243/39769473-26e1fe02-52a1-11e8-9e68-8a2de841d2f3.png)

When the minibatch size is changed to 2, the net stops learning.
![screen shot 2018-04-05 at 10 54 12 am](https://user-images.githubusercontent.com/7873243/39769490-396ef84a-52a1-11e8-901f-771afebf359f.png)

The hyperparameters used are the same:

>          Net Summary
> ---------------------------------
> patch_size     = 3
> num_epochs     = 10
> minibatch_size = 1
> learning_rate  = 0.0001
> Cost Function  = dice_hard_coe
> Filters per layer = [16 32 64 128 ]

In this tensorboard chart, you can see that the loss for batch size 2 is not minimizing.
![screen shot 2018-04-05 at 10 54 26 am](https://user-images.githubusercontent.com/7873243/39769649-a1e6c01a-52a1-11e8-8ada-f30ec093b8b4.png)

Any suggestions will be highly appreciated.

**Comment by CataRay** [comment]
Solved: Relu in CONV1 was skipped by mistake

![screen shot 2018-05-09 at 3 39 21 pm](https://user-images.githubusercontent.com/7873243/39843271-294a7940-539f-11e8-8330-ce06aafcc378.png)
